rhel7/profiles/stig.profile

```diff
@@ -273,6 +273,7 @@ selections:
     - sshd_enable_x11_forwarding
     - tftpd_uses_secure_mode
     - package_xorg-x11-server-common_removed
+    - xwindows_runlevel_target
```

Comment:
Does this rule exist in the STIG?

Reply:
Per: `U_Red_Hat_Enterprise_Linux_7_STIG_V2R6_Manual-xccdf.xml`, there isn't.

Reply:
The rule has been removed from rule selection list